Frontend/src/components/institutions/InstitutionTeamList/index.jsx

```diff
@@ -8,9 +8,12 @@ function InstitutionTeamList({ onLeft, institution }) {
   const [teams, setTeams] = useState([]);
   const [teamsLoaded, setTeamsLoaded] = useState(false);
 
-  useEffect(async () => {
-    setTeams(await TeamService.getInstitutionTeams(institution.id));
-    setTeamsLoaded(true);
+  useEffect(() => {
+    TeamService.getInstitutionTeams(institution.id)
+      .then((newTeams) => {
+        setTeams(newTeams);
+        setTeamsLoaded(true);
```

Comment:
What if the request fails? Then you will get the Spinner forever. We have done it differently but you can just add a `.finally(()=> setTeamsLoaded(true))` & also remove it from `.then clause`